.circleci/config.yml

```diff
@@ -7,10 +7,13 @@ jobs:
       - checkout
       - run:
           name: Install build tools
-          command: sudo apt-get update && sudo apt-get install build-essential cpio squashfs-tools debootstrap realpath pyflakes3
+          command: sudo apt-get update && sudo apt-get install build-essential cpio squashfs-tools debootstrap realpath pyflakes3 shellcheck
       - run:
           name: Run pyflakes3
           command: pyflakes3 .
+      - run:
+          name: Run shellcheck
+          command: tools/shellcheck-all.sh
```

Comment:
Any reason this (850f63f41f9eef64bfa1c9345b5053ca318bd63e) isn't part of the last commit (f3d45368324947d523d65f33ad2948b33b969586)?

Reply:
Because I intended to order the "fix shellcheck errors" step before this one

Reply:
And indeed I did, even before the rebase.  I still think whatever interface you're using to view the commits is not giving you them in topological order.